content/docs/reference/templates/functions.md

```diff
@@ -1241,6 +1242,17 @@ the triggering message and ping the author of that message, but suppress the res
 {{ sendMessage nil $message }}
 ```
 
+Example of a message forward:
+
+```yag
+{{ sendMessage nil (complexMessage
+    "forward" (sdict
+        "channel" .Channel.ID
+        "message" <messageID>
```

Comment:
Not sure if we want placeholders in examples--ideally they should be valid code as is. Can we use `.Message.ID` instead?